ci-operator/step-registry/baremetalds/e2ev2/baremetalds-e2ev2-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: baremetalds-e2ev2
+  steps:
+    pre:
+      - chain: baremetalds-ipi-pre
+    test:
+      - ref: baremetalds-e2e-conf
+      - ref: baremetalds-e2ev2-test
```

Comment:
nit: it's sufficient the e2ev2 step, the workflow could be defined directly at the job level